### PR TITLE
Add wrinkled anatomical entity DOS-DP pattern draft

### LINKED ISSUE #858: Add wrinkled anatomical entity pattern template
[wrinkled PATO:0001810](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0001810&lang=en&viewMode=PreferredRoots&siblings=false)

1. [x] add `wrinkledAnatomicalEntity.yaml` DOS-DP pattern draft for community review
2. [x] get feedback
3. [ ] fix or improve pattern based on feedback

* Examples:
    * [MP:0011205	excessive folding of visceral yolk sac](https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMP_0011205)
    * [MP:0021107	retina fold](https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMP_0021107)
    * [HP:0008052	Retinal fold](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0008052)

### PR DESCRIPTION
**Pull request by rays22**
for community review.

If applied, this commit will fix #858.

### CODE REVIEW

File: src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+equivalentTo:
+  text: "'has_part' some (
+    'wrinkled' and
+    ('characteristic_of' some %s) and
+    ('has_modifier' some 'abnormal')
+    )"
+  vars:
+    - anatomical_entity
+...
```
Comment (matentzn) [inline src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml:61]: I am really worried about these three dots at the end of every pattern.. Why are these there?

File: src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/wrinkledAnatomicalEntity.yaml
+
+description: "Use this phenotype pattern when anatomical entity is
+  marked by one or more creases in a normally smooth surface. Consider
+  the 'foldedAnatomicalEntity.yaml' phenotype pattern when the
+  anatomical when an anatomical entity has morphological abnormalities
+  involving folding all or any part of it over another connected part."
+
+#  examples:
```
Comment (matentzn) [inline src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml:12]: why are the examples commented out?

Comment (sbello) [reply src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml:12]: add example 
wrinkled skin MP:0001211
ZP:0015987 http://purl.obolibrary.org/obo/ZP_0015987

examples were commented out until review

File: src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml (hunk 1 above)
Comment (sbello) [reply src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml:61]: I do not know, will ask Ray

**Review by sbello — CHANGES_REQUESTED** [review]

File: src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml (hunk 1 above)
Comment (rays22) [reply src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml:61]: > I am really worried about these three dots at the end of every pattern.. Why are these there?

They are [yaml 1.2.0 end of document markers](https://yaml.org/spec/1.2.0/#id2599514).
I get a warning from [yamlint](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_end) if I leave them out.

File: src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml (hunk 2 above)
Comment (rays22) [reply src/patterns/dosdp-dev/wrinkledAnatomicalEntity.yaml:12]: > why are the examples commented out?

Examples are not yet implemented in DOS-DP tools and thus the validation fails unless I comment them out. I will come back and fix them after the examples section gets implemented in DOS-DP tools.

**Pull request merged**